app/helpers/physical_chassis_helper/textual_summary.rb

```diff
@@ -0,0 +1,113 @@
+module PhysicalChassisHelper::TextualSummary
```

Comment:
Please, add tests for this helper. There should be enough example both simple and complex, many from your team. Thx!